src/lib/storage.js

```diff
@@ -507,7 +509,14 @@ class Storage implements IStorageHandler {
           if (err) {
             return callback(err);
           }
-          return callback(null, packageJsonLocal, upLinksErrors);
+          async.map(
```

Comment:
I'd prefer do not use `async.map()` anymore https://github.com/verdaccio/verdaccio/issues/1225

Reply:
Fixed